scripts/install.sh

```diff
@@ -931,6 +935,13 @@ echo 'Changing permissions for config file and storage'
 chmod 440 "${CONFIG_PATH}"
 chmod -R 750 "${FILE_STORAGE}"
 
+echo 'Changing permissions for user env directory'
+chmod 440 "${USER_ENV_DIRECTORY}"
+chmod g+s "${USER_ENV_DIRECTORY}"
+
+echo 'Changing ownership for user env directory'
+chown -R "${SYSTEM_USER}":"${SYSTEM_USER}" "${USER_ENV_DIRECTORY}" 
```

Comment:
I like this separation (from conf.d), OT configuration can remain visible to most system users while sensitive values are obfuscated/protected.

Reply:
@jspaleta the system user doesn't yet exist.